metrics/system_tag.go

```diff
@@ -7,61 +7,70 @@ import (
 	"strings"
 )
 
-// SystemTagSet is a bitmask that is used to keep track
-// which system tags should be included with which metrics.
-//go:generate enumer -type=SystemTagSet -transform=snake -trimprefix=Tag -output system_tag_set_gen.go
-type SystemTagSet uint32
+// SystemTag values are bit-shifted identifiers of all of the various system tags that k6 has.
+//go:generate enumer -type=SystemTag -transform=snake -trimprefix=Tag -output system_tag_gen.go
+type SystemTag uint32
+
+// SystemTagSet is a bitmask that is used to keep track which system tags should
+// be included in metric Samples.
+type SystemTagSet SystemTag
 
 // Default system tags includes all of the system tags emitted with metrics by default.
 const (
-	TagProto SystemTagSet = 1 << iota
+	TagProto SystemTag = 1 << iota
 	TagSubproto
 	TagStatus
 	TagMethod
 	TagURL
+	TagRawURL // non-indexable
 	TagName
 	TagGroup
 	TagCheck
-	TagError
+	TagError // non-indexable
 	TagErrorCode
 	TagTLSVersion
 	TagScenario
 	TagService
 	TagExpectedResponse
 
 	// System tags not enabled by default.
-	TagIter
-	TagVU
+	TagIter // non-indexable
+	TagVU   // non-indexable
 	TagOCSPStatus
-	TagIP
+	TagIP // non-indexable
```

Comment:
Hmm, actually, I am not sure that IP should be non-indexable :thinking: it is not enabled by default, and generally it shouldn't be too high cardinality, right? Even a geo-distributed CDN will probably only have dozens of unique IPs in a load test :thinking:

Reply:
I think we can start as a non-indexable and maybe change in the future. I think the reverse could be more complicated. In any case, the user can force to have it indexed if it's required, right?

Reply:
No, you currently can't force a system tag to be indexed. It would probably be fairly simple to make that configurable, but I am not sure we should, it might lead to more issues than it solves... :thinking:

Reply:
FYI my intention is to roll back this change and leave the IP address as an indexable (i.e. normal) tag

Reply:
Resolved by [this force-pushed fixup](https://github.com/grafana/k6/compare/d3d0eb8af4b14a169143126a0f839f1f6833b153..c8bfdd7f722411ed6846fe41d626ab07f618712f) of the original commit.